samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -876,10 +877,18 @@ mfxStatus CEncodingPipeline::AllocFrames()
     MSDK_ZERO_MEMORY(VppRequest[0]);
     MSDK_ZERO_MEMORY(VppRequest[1]);
 
+    mfxU16 tmpTargetKbps = m_mfxEncParams.mfx.TargetKbps;
+
     // Querying encoder
     sts = GetFirstEncoder()->Query(&m_mfxEncParams, &m_mfxEncParams);
     MSDK_CHECK_STATUS(sts, "Query (for encoder) failed");
 
+    if (MFX_WRN_INCOMPATIBLE_VIDEO_PARAM == sts && tmpTargetKbps != m_mfxEncParams.mfx.TargetKbps)
+    {
+        msdk_printf(MSDK_STRING("WARNING: Target bitrate was changed from %d kbps to %d kbps due to default limitations.\n"), tmpTargetKbps, m_mfxEncParams.mfx.TargetKbps);
```

Comment:
"due to default limitations" - IMHO this is unclear. Better to re-phrase or even remove.